composer.json

```diff
@@ -34,7 +34,7 @@
         "phpunit/phpunit": "^7.0.1",
         "zendframework/zend-coding-standard": "~1.0.0",
         "zendframework/zend-expressive-helpers": "^5.0.0alpha3",
-        "zendframework/zend-hydrator": "^2.3.1",
+        "zendframework/zend-hydrator": "^2.3.1 || ^3.0",
```

Comment:
Would it make sense to just bump this to `^3.0` instead? I don't see a reason for supporting both releases.

Reply:
@Ocramius I think we NEED support BOTH versions as `zend-expressive-hal` supports PHP 7.1 and `zend-hydrator` v3 supports only PHP 7.2.

Reply:
See https://github.com/zendframework/zend-expressive-hal/pull/51#issuecomment-446246085 - I'm in the "drop 7.1 camp". This just means that approval/merge goes through somebody else though ;-)

Reply:
Ok, missed the discussion below :) Thanks 👍